### PR TITLE
Wagtail 2.8

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/4403

### CODE REVIEW

File: dockerfiles/Dockerfile.python [hunk 1]
```diff
@@ -15,7 +15,7 @@ WORKDIR /app/
 RUN pip install pipenv
 
 # Copy Pipfiles in the container
-COPY Pipfile Pipfile.lock ./
+COPY Pipfile.*  ./
```
Comment (Pomax) [inline dockerfiles/Dockerfile.python]: this fixes the problem that if you delete your local `Pipfile.lock`, and then try to run `docker-catchup` and `docker-new-env`, those commands will error out.

Comment (cadecairos) [reply dockerfiles/Dockerfile.python]: nice workaround

Comment (Pomax) [reply dockerfiles/Dockerfile.python]:
```suggestion
COPY Pipfile*  ./
```

File: dockerfiles/Dockerfile.python [hunk 2]
```diff
@@ -15,7 +15,7 @@ WORKDIR /app/
 RUN pip install pipenv
 
 # Copy Pipfiles in the container
-COPY Pipfile Pipfile.lock ./
+COPY Pipfile*  ./
```
Comment (Pomax) [inline dockerfiles/Dockerfile.python]:
```suggestion
COPY Pipfile* ./
```

**Pull request merged**